upgrading/migrating_etcd.adoc

```diff
@@ -103,7 +103,7 @@ The following procedure describes the steps required to successfully migrate the
 cluster (implemented as part of the Ansible etcd migration playbook).
 
 . +++<b>Create an etcd backup.</b>+++ See
-xref:../../admin_guide/backup_restore.adoc#cluster-backup[Backup and Restore]
+xref:../admin_guide/backup_restore.adoc#cluster-backup[Backup and Restore]
```

Comment:
When you click thru, the Backup and Restore topic is currently missing a lot of content in the build. Not sure what's up.

Reply:
Hmm, looks like a problem in master.